src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -67,33 +68,21 @@ private void PopulateView()
 
                     isLoading = true;
 
-                    Client.GetCurrentUser(user => {
-                        if (user == null)
+                    Client.LoadKeychain(hasKeys => {
+                        if (!hasKeys)
                         {
-                            Logger.Warning("Unable to get current user");
                             return;
                         }
 
-                        owners = new[] { user.Login };
-                        username = user.Login;
+                        username = keychainConnections.First().Username;
 
-                        Logger.Trace("GetOrganizations");
+                        Client.GetOrganizations(organizations => {
 
-                        Client.GetOrganizations(organizations =>
-                        {
-                            if (organizations == null)
-                            {
-                                Logger.Warning("Unable to get list of organizations");
-                                return;
-                            }
-
-                            Logger.Trace("Loaded {0} organizations", organizations.Count);
-
-                            var organizationLogins = organizations
+                            var organizationLogins = (organizations ?? Enumerable.Empty<Organization>())
```

Comment:
Instead of returning null from `GetOrganizations`, return an empty enumerable so the caller doesn't have to guess whether it can use the return value.